packages/round-manager/src/index.tsx

```diff
@@ -34,6 +35,17 @@ import { BulkUpdateGrantApplicationProvider } from "./context/application/BulkUp
 // Initialize datadog
 initDatadog();
 
+// Intialize TagManager
+if (process.env.REACT_APP_TAG_MANAGER) {
+  TagManager.initialize({
+    gtmId: `${process.env.REACT_APP_TAG_MANAGER}`,
+    events: {
+      "gtm.start": new Date().getTime(),
+      "event": "gtm.js"
+    }
```

Comment:
Gerrit to check and ack if this is right format and nothing more has to be added

Reply:
Checked both GE and RM locally and everything appears to be firing 100%.